packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -665,6 +665,9 @@ private void startPreview() throws CameraAccessException {
 
       Surface previewSurface = new Surface(surfaceTexture);
       surfaces.add(previewSurface);
+      if(surfaces == null || imageReader == null){
+        return;
+      }
```

Comment:
Why would they be null here? Are we doing something wrong in how we obtain them?

Reply:
I also don't know yet why they would be null here. this is related to: https://github.com/flutter/flutter/issues/19595

Reply:
I think the `camera.open()` is not being called? 
https://github.com/CircleCurve/plugins/blob/1ceefbcc1f0e823bd2640fa3decbaa998b2be163/packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java#L95-L97